src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -102,12 +112,99 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->convertController($route);
             }
 
-            $helper->describe($output, $routes, array(
-                'format' => $input->getOption('format'),
-                'raw_text' => $input->getOption('raw'),
-                'show_controllers' => $input->getOption('show-controllers'),
-            ));
+            if ('txt' === $input->getOption('format')) {
+                $this->displayRouteCollectionInformation($routes, $input->getOption('show-controllers'), $output);
+            } else {
+                $helper->describe($output, $routes, array(
+                    'format' => $input->getOption('format'),
+                    'raw_text' => $input->getOption('raw'),
+                    'show_controllers' => $input->getOption('show-controllers'),
+                ));
+            }
+        }
+    }
+
+    private function displayRouteInformation($name, Route $route, OutputInterface $output)
+    {
+        $requirements = $route->getRequirements();
+
+        $output->table(
```

Comment:
there is no ``->table()`` method in the OutputInterface. so the typehint looks weird